set.mm

```diff
@@ -404277,34 +404375,34 @@ According to Wikipedia ("Path (graph theory)",
   $}
 
   ${
-    $( TODO-AV: ` H = ( F |` ( 0 ..^ N ) ) ` could be replaced by
-                ` H = ( F prefix N ) ` ! $)
     trlres.v $e |- V = ( Vtx ` G ) $.
     trlres.i $e |- I = ( iEdg ` G ) $.
     trlres.d $e |- ( ph -> F ( Trails ` G ) P ) $.
     trlres.n $e |- ( ph -> N e. ( 0 ..^ ( # ` F ) ) ) $.
-    trlres.h $e |- H = ( F |` ( 0 ..^ N ) ) $.
+    trlres.h $e |- H = ( F prefix N ) $.
     $( Lemma for ~ trlres .  Formerly part of proof of ~ eupthres .
        (Contributed by Mario Carneiro, 12-Mar-2015.)  (Revised by Mario
-       Carneiro, 3-May-2015.)  (Revised by AV, 6-Mar-2021.) $)
+       Carneiro, 3-May-2015.)  (Revised by AV, 6-Mar-2021.)  (Revised by AV,
+       30-Nov-2022.) $)
```

Comment:
Maybe two "revised" tags in a row are unnecessary, especially if the revisions are not explained.

Reply:
OK, I will add an explanation...